src/Microsoft.AspNetCore.Antiforgery/ServiceCollectionExtensions.cs

```diff
@@ -10,9 +10,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up antiforgery services in an <see cref="IServiceCollection" />.
+    /// </summary>
     public static class ServiceCollectionExtensions
     {
-        public static IServiceCollection AddAntiforgery(this IServiceCollection services)
+        /// <summary>
+        /// Adds antiforgery services to the specified <see cref="IServiceCollection" />.
```

Comment:
Forget my comment, I don't have a strong take between using paramref vs see in this case, is there a clear advantage on doing that? I'm saying it because it's not trivial to go through all the repositories changing this.